abci/types/application.go

```diff
@@ -117,3 +117,13 @@ func (BaseApplication) FinalizeBlock(_ context.Context, req *RequestFinalizeBloc
 		TxResults: txs,
 	}, nil
 }
+
```

Comment:
If we keep the comments, can we add the following?
`// TODO CHECK HEIMDALL-V2 check implementations of BeginSideBlock and DeliverSideTx`

Reply:
Won't need them anymore.